02_Util/src/index.ts

```diff
@@ -2,7 +2,7 @@
 // Copyright Contributors to the CitrineOS Project
 //
 // SPDX-License-Identifier: Apache 2.0
-
+import "reflect-metadata";
```

Comment:
this has to be early on in every package for tsyringe as well as our annotations and will probably be needed by other things so I added to all modules